src/main/java/org/broadinstitute/hellbender/tools/exome/HetPulldownCalculator.java

```diff
@@ -156,18 +160,18 @@ public Pulldown getTumor(final File tumorBAMFile, final IntervalList normalHetIn
      * @return                  Pulldown of heterozygous SNP sites in 1-based format
      */
     private Pulldown getHetPulldown(final File bamFile, final IntervalList snpIntervals, SampleType sampleType,
-                                           final double hetAlleleFraction, final double pvalThreshold) {
+                                    final double hetAlleleFraction, final double pvalThreshold) {
         try (final SamReader bamReader = SamReaderFactory.makeDefault().open(bamFile);
-             final ReferenceSequenceFileWalker refWalker = new ReferenceSequenceFileWalker(this.refFile)
-        ) {
+             final ReferenceSequenceFileWalker refWalker = new ReferenceSequenceFileWalker(this.refFile)) {
             if (bamReader.getFileHeader().getSortOrder() != SAMFileHeader.SortOrder.coordinate) {
                 throw new UserException.BadInput("BAM file " + bamFile.toString() + " must be coordinate sorted.");
             }
 
             final Pulldown hetPulldown = new Pulldown(bamReader.getFileHeader());
 
+            final int snpTotalNumber = snpIntervals.size();
```

Comment:
As long as you're making this more human-readable, may as well name the variable `totalNumberOfSnps`.